src/zebrafish_toxicology_atlas_schema/schema/zebrafish_toxicology_atlas_schema.yaml

```diff
@@ -15,236 +15,261 @@ prefixes:
   schema: http://schema.org/
   PATO: http://purl.obolibrary.org/obo/PATO_
   example: https://example.org/
+  CHEBI: http://purl.obolibrary.org/obo/CHEBI_
+  CAS: https://commonchemistry.cas.org/detail?cas_rn=
+  ECTO: http://purl.obolibrary.org/obo/ECTO_
+  EXO: http://purl.obolibrary.org/obo/EXO_
 default_prefix: zebrafish_toxicology_atlas_schema
 default_range: string
 
 imports:
   - linkml:types
 
 classes:
-  NamedThing:
-    abstract: true
-    description: A named entity.
+
+  Study:
+    description: A toxicology study containing experiments.
     slots:
+      - has_experiments
+      - publication_id
```

Comment:
We also need: 
- submitter (or contributor) - orcid - >1
- lab (where the experiment came from) - ZFiN ID for labs